DependencyInjection/Configuration.php

```diff
@@ -17,6 +17,7 @@ final class Configuration implements ConfigurationInterface
     public function getConfigTreeBuilder()
     {
         $treeBuilder = $this->getWrappedTreeBuilder('trikoder_oauth2');
+        /** @var NodeDefinition $rootNode */
```

Comment:
@Allypost Just noticed this, I think a better solution would be to fix the return type of `getWrappedTreeBuilder`, should be `TreeBuilder`.

Reply:
@HypeMC Fixed.